packages/server/src/common/operations/compound-operation-handler.ts

```diff
@@ -15,20 +15,29 @@
  ********************************************************************************/
 import { CompoundOperation, MaybePromise, Operation } from '@eclipse-glsp/protocol';
 import { inject, injectable } from 'inversify';
+import { Command, CompoundCommand } from '../command/command';
 import { OperationActionHandler } from './operation-action-handler';
 import { OperationHandler } from './operation-handler';
 import { OperationHandlerRegistry } from './operation-handler-registry';
 
 @injectable()
-export class CompoundOperationHandler implements OperationHandler {
+export class CompoundOperationHandler extends OperationHandler {
+    @inject(OperationHandlerRegistry)
+    protected operationHandlerRegistry: OperationHandlerRegistry;
+
     operationType = CompoundOperation.KIND;
 
-    @inject(OperationHandlerRegistry) protected operationHandlerRegistry: OperationHandlerRegistry;
+    async createCommand(operation: CompoundOperation): Promise<Command | undefined> {
```

Comment:
Since we are no longer calling `executeNestedOperation` I believe we can simply remove that method.

Reply:
Good point. Is removed with 41f9759